feedback-module/src/services/requestService.js

```diff
@@ -11,14 +11,14 @@ function requestService(
     .post(`${process.env.REACT_APP_BACKEND_ENDPOINT}/api/${apiEndpoint}`, obj)
     .then((res) => {
       console.log(res);
-      onSuccess();
-      setFailedRequest(false);
+      onSuccess && onSuccess();
+      setFailedRequest && setFailedRequest(false);
```

Comment:
Do we still need to check if onSuccess and setFailedRequest exist because now don't we always pass these functions in?

Reply:
Yeah as of now we always pass the functions in, but I decided to add this check in case someone uses this function somewhere else later and decides not to pass in those arguments for whatever reason.

Reply:
Ok but I feel like it should just be implied that every time we make a network call we check to see if it went through

Reply:
Yeah true, this was just an extra measure just in case, but if you think we don't need it, I can just remove it. If we do that, I think we should add something in the documentation or comments specifying that all of those methods are required though.

Reply:
Hmm I guess we can keep it but maybe still in the documentation we say that it should be there except for extreme circumstances